applications/fax/src/fax_worker.erl

```diff
@@ -801,10 +801,9 @@ fetch_document_from_url(JObj) ->
                  ,{"User-Agent", wh_json:get_string_value(<<"user_agent">>, FetchRequest, wh_util:to_list(node()))}
                  ,{"Content-Type", wh_json:get_string_value(<<"content_type">>, FetchRequest, <<"text/plain">>)}
                 ]),
-    HTTPOptions = [{'response_format', 'binary'}],
     Body = wh_json:get_string_value(<<"content">>, FetchRequest, ""),
     lager:debug("making ~s request to '~s'", [Method, Url]),
-    ibrowse:send_req(Url, Headers, Method, Body, HTTPOptions).
+    kz_http:req(Method, Url, Headers, Body).
```

Comment:
For flexibility for the times that we don't know the method yet or we should decide on using which method based on the request or user configurations.